wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/PartitionedSchema.kt

```diff
@@ -111,6 +131,36 @@ internal fun Schema.partition(modules: Map<String, Module>): PartitionedSchema {
   return PartitionedSchema(partitions, warnings, errors)
 }
 
+/** Return a copy of this type with all possible type references removed. */
+private fun Type.asStub(): Type = when {
```

Comment:
do we do top-level expressions elsewhere? Should we?

Reply:
I don't know many arguments against them, especially not private ones. These two are used by another top-level function that landed in the previous PR so for now I'll keep.